clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -1762,6 +1762,29 @@ def err_user_defined_msg_constexpr : Error<
   "%sub{subst_user_defined_msg}0 must be produced by a "
   "constant expression">;
 
+// Type traits explanations
+def note_unsatisfied_trait : Note<"%0 is not %enum_select<TraitName>{"
+                                  "%TriviallyRelocatable{trivially relocatable}"
+                                  "}1">;
+
+def note_unsatisfied_trait_reason
+    : Note<"because it "
```

Comment:
My word... I'm VERY glad we have `enum_select` :D

Reply:
Yeah, in these cases it's super useful, thanks :D